### ISSUE TITLE
Find all files - case insensitive search is broken

### ISSUE THREAD

**Issue by fieryorc** [Microsoft/vscode#65568]
Issue Type: <b>Bug</b>

Press - Ctrl + Shift + F to open find all files
Enter search string

It only searches files matching exact case even though the case sensitive button is toggled off. Regardless of the button state, the search always returns only case sensitive match.

VS Code version: Code 1.30.1 (dea8705087adb1b5e5ae1d9123278e178656186a, 2018-12-18T18:07:32.870Z)
OS version: Linux x64 4.18.10-1rodete2-amd64

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E5-1650 v3 @ 3.50GHz (6 x 1619)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: unavailable_software<br>video_encode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 2, 1|
|Memory (System)|31.39GB (2.36GB free)|
|Process Argv|-n .|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
solargraph|cas|0.18.0
gotemplate|cas|0.3.0
xml|Dot|2.4.0
tslint|eg2|1.0.42
python|ms-|2018.12.1
Go|ms-|0.8.0
vscode-docker|Pet|0.4.0
typescript-hero|rbb|3.0.0
java|red|0.36.0
vscode-icons|rob|8.0.0
vscode-asciidoc|sta|0.0.3
code-spell-checker|str|1.6.10
vscode-java-debug|vsc|0.16.0
vscode-java-dependency|vsc|0.3.0
vscode-java-pack|vsc|0.5.0
vscode-java-test|vsc|0.12.0
vscode-maven|vsc|0.12.1
gitblame|wad|2.6.3
clang-format|xav|1.8.0
vscode-proto3|zxh|0.2.2

</details>

**Comment by roblourens** [comment]
Sounds like you might have `search.smartCase` turned on?

**Comment by fieryorc** [comment]
Yes, it is on. But it started happening only after recent update. Is this turned on for "Find all files" recently?

Either way, I tested with all lower case and it works as expected. However, it would be nice to have some indicator of some sort or disable the case sensitivity button if smart search is enabled.

**Comment by roblourens** [comment]
It should not be turned on by default. Seems that you turned probably it on by mistake. I don't have any UI for it currently because it's a little-used feature.
